Source/Libraries/openXDA.Model/Emails/BaseEmailType.cs

```diff
@@ -0,0 +1,39 @@
+﻿//******************************************************************************************************
+//  IEmailType.cs - Gbtc
```

Comment:
Convention is to put the word `Base` at the end of the class name. The file will have to be renamed as well.

```suggestion
//  EmailTypeBase.cs - Gbtc
```